sse2neon.h

```diff
@@ -8828,9 +8917,14 @@ FORCE_INLINE __m128i _mm_aeskeygenassist_si128(__m128i a, const int rcon)
 // for more details.
 FORCE_INLINE __m128i _mm_aesenc_si128(__m128i a, __m128i b)
 {
+#ifdef _MSC_VER
```

Comment:
This shouldn't be strictly necessary. It should be possible to tweak the code in the #else block slightly to work with MSVC by avoiding GNU extensions (e.g. replacing operator ^ with veorq_u8).

Reply:
ACK, fixed in next commit